src/ui/map.js

```diff
@@ -2830,12 +2830,13 @@ class Map extends Camera {
      *     }, 'hover');
      * });
      *
-    */
+     */
     removeFeatureState(feature: { source: string; sourceLayer?: string; id?: string | number; }, key?: string): this {
         this.style.removeFeatureState(feature, key);
         return this._update();
     }
 
+
```

Comment:
More than 1 blank line is not allowed in the linter rules, could you please remove it?

Reply:
Hello @stepankuzmin,

I've gone ahead and removed the extra blank line as requested. Could you please review the changes and let me know if any further modifications are needed? If everything looks good, I'd appreciate it if you could merge the PR.

Thank you,
Muditya Raghav